.php_cs.dist

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
How config path is determined:
- path from CLI argument
- path of what you are fixing (`php-cs-fixer fix foo/` -> `foo/.php_cs(.dist)`)
- current working dir (`php-cs-fixer fix foo/` -> `.php_cs(.dist)`)

to work the best when one may want to run tool providing only subfolders, like
`php-cs-fixer fix foo/`
and then
`php-cs-fixer fix bar/`
the best DX is when the config file is on `cwd` level, so root of the project